lib.py

```diff
@@ -119,26 +108,15 @@ async def build_code(author_name: str, author_id: int, tmp_dir: str) -> bool:
     """
     logger.info("Running container to build code for %s", author_id)
     try:
-        loop = asyncio.get_event_loop()
-        out = await loop.run_in_executor(
-            None,
-            functools.partial(
-                doc.containers.run,
-                settings.docker.container_ref,
-                "timeout --kill-after=5s 30s cargo build",
-                remove=True,
-                stdout=True,
-                mem_limit="8g",
-                network_mode="none",
-                # Don't mount the inputs directory here for defense-in-depth
-                volumes={
-                    os.path.join(tmp_dir, "src"): {"bind": "/app/src", "mode": "rw"},
-                    os.path.join(tmp_dir, "benches"): {"bind": "/app/benches", "mode": "rw"},
-                    os.path.join(tmp_dir, "target"): {"bind": "/app/target", "mode": "rw"},
-                },
-            ),
+        out = await run_cmd(
+            "timeout --kill-after=5s 30s cargo build",
```

Comment:
lemmie just
```suggestion
            "timeout --kill-after=5s 90s cargo build --release",
```
this should make it not full rebuild in the `cargo criterion` section